SPDX-FileCopyrightText: 2005-2020 Free Software Foundation, Inc.

SPDX-License-Identifier: GPL-3.0-or-later

diff --git a/modules/tempname b/modules/tempname
index 570ea54..f1be8ff 100644
--- a/modules/tempname
+++ b/modules/tempname
@@ -13,6 +13,8 @@
 gettimeofday
 lstat
 mkdir
+randint
+stdbool
 stdint
 sys_stat
 sys_time
